src/classes/RD2_RecurringDonations_TDTM.cls

```diff
@@ -283,7 +283,7 @@ public class RD2_RecurringDonations_TDTM extends TDTM_Runnable {
 
         Map<Id, List<RecurringDonationSchedule__c>> oldSchedules = new Map<Id, List<RecurringDonationSchedule__c>>();
         for (RecurringDonationSchedule__c oldSchedule : [
-                SELECT RecurringDonation__c, EndDate__c FROM RecurringDonationSchedule__c
+                SELECT RecurringDonation__c, StartDate__c, EndDate__c FROM RecurringDonationSchedule__c
```

Comment:
I believe this method moved to the schedule service in #4800. I suspect it's not a problem, other than dealing with the eventual merge conflict in this class.